pydrive2/drive.py

```diff
@@ -47,3 +47,4 @@ def GetAbout(self):
         :returns: A dictionary of Google Drive information like user, usage, quota etc.
         """
         return self.auth.service.about().get().execute(http=self.http)
+        
```

Comment:
minor: let's not do these unrelated changes please?

Reply:
solved in [d39458c](https://github.com/iterative/PyDrive2/pull/188/commits/d39458cdce6b55c416fa13d64e0cdeeef4eae0f7)